klog_test.go

```diff
@@ -753,3 +753,126 @@ func TestKRef(t *testing.T) {
 		})
 	}
 }
+
+var testData = []struct {
```

Comment:
As a global variable this is poorly named.  

The way I would write these tests is:

1) test the heck out of kvListFormat() with this data as the test cases, defined in the test function.

2) Simple test of InfoS and ErrorS that proves that it correctly formats a few simple cases.

alternately, if you want to really exercise end-to-end, put this test data into a function or at least name it better.